src/Helper/SplitStreet.php

```diff
@@ -57,9 +94,21 @@ class SplitStreet
      */
     public static function splitStreet(string $fullStreet, string $local, string $destination): FullStreet
     {
-        $translateBoxSeparator = str_ireplace(self::BOX_TRANSLATION_POSSIBILITIES, ' ' . self::BOX_NL, $fullStreet);
-        $fullStreet            = trim(preg_replace('/(\r\n)|\n|\r/', ' ', $translateBoxSeparator));
-        $regex                 = self::getRegexByCountry($local, $destination);
+        foreach (self::NUMBER_SUFFIX_ABBREVIATION as $from => $to) {
```

Comment:
Dit moet met `if ($destination === AbstractConsignment::CC_NL) {`
Het is alleen voor NL

Reply:
Done